src/js/modules/infragistics.ui.editors.js

```diff
@@ -738,48 +876,77 @@
 			}
 		},
 		field: function () {
-			/* Gets the visual editor element.
+			/* Gets the visual editor (input) element.
+			```
+			$(".selector").%%WidgetName%%("field");
+			```
 			returnType="$" The visual editor element. */
 			return this._editorInput;
 		},
 		editorContainer: function () {
-			/* Gets reference to jquery object which is used as top/outer element of igEditor.
+			/* Gets reference to jquery object which is used as top/outer element of th editor.
+			```
+			$(".selector").%%WidgetName%%("editorContainer");
+			```
 			returnType="$" The container editor element. */
 			return this._editorContainer;
 		},
 		hasFocus: function () {
 			/* Checks if editor has focus.
+			```
+				$(".selector").%%WidgetName%%("hasFocus");
+			```
 				returnType="bool" Returns if the editor is focused or not. */
 			return this._focused;
 		},
 		setFocus: function (delay) {
-			/* Set focus to editor with delay.
+			/* Set focus to the editor with delay.
+				```
+				$(".selector").%%WidgetName%%("setFocus", 200);
+				```
 				paramType="number" optional="true" The delay before focusing the editor. */
 			this._setFocusDelay(delay);
 		},
 		hide: function () {
-			/* Hides editor. */
+			/* Hides editor.
+			```
+			$(".selector").%%WidgetName%%("hide");
+			```
+			*/
 			this._editorContainer.hide();
 		},
 		show: function () {
-			/* Shows editor. */
+			/* Shows editor.
+			```
+			$(".selector").%%WidgetName%%(("show");
+			```
+			*/
 			this._editorContainer.show();
 		},
 		validator: function () {
-			/* Gets reference to igValidator used by the editor.
-				returnType="object" Returns reference to igValidator or null. */
+			/* Gets reference to [igValidator](ui.igvalidator) used by the editor.
+				```
+					var validator = $(".selector").%%WidgetName%%("validator");
+				```
+				returnType="object" Returns reference to [igValidator](ui.igvalidator) or null. */
 			return this._validator;
 		},
 		isValid: function () {
 			/* Checks if value in editor is valid. Note: This function will not trigger automatic notifications.
```

Comment:
Checks if the value in the editor is valid. 

I don't understand what the second part means, about the automatic notifications.